src/Translatable/Query/TreeWalker/TranslationWalker.php

```diff
@@ -457,8 +458,11 @@ private function getCastedForeignKey($component, $typeFK, $typePK)
             return $component;
         }
 
-        // try to look at postgres casting
-        if ($this->platform instanceof PostgreSqlPlatform) {
+        // doctrine/dbal 3.x drops support for PostgreSQL 9.3 and earlier, the below check prefers the 
+        // `PostgreSQL94Platform` class as it is available in both major DBAL versions and falls back to the 
```

Comment:
Please, check the CI status.
```suggestion
        // doctrine/dbal 3.x drops support for PostgreSQL 9.3 and earlier, the below check prefers the
        // `PostgreSQL94Platform` class as it is available in both major DBAL versions and falls back to the
```